sirepo/feature_config.py

```diff
@@ -143,15 +144,15 @@ def b(msg, dev=False):
             display_test_boxes=b('Display test boxes to visualize 3D -> 2D projections'),
         ),
     )
-    i = _cfg.proprietary_sim_types.intersection(_cfg.default_proprietary_sim_types)
+    i = _cfg.proprietary_sim_types.intersection(_cfg.proprietary_with_moderation_prompt_sim_types)
     assert not i, \
-        f'{i}: cannot be in proprietary_sim_types and default_proprietary_sim_types'
+        f'{i}: cannot be in proprietary_sim_types and proprietary_with_moderation_prompt_sim_types'
```

Comment:
I think these are now orthogonal. proprietary_sym_types control where the code resides. moderated_sym_types control the authorization. You might not want to have proprietary_sym_types that are moderated (ITAR codes, for example), and vice versa (jupyter).

Reply:
84a51a4c4d292b403b922c73411c9bad58de76bf